src/components/FieldArray/FieldArray.css

```diff
@@ -0,0 +1,19 @@
+@import '../../marketplace.css';
+
+.root {
+}
+
+.formRow {
+  display: flex;
+  justify-content: space-between;
+  flex-shrink: 0;
+  width: 100%;
+  margin-bottom: 24px;
+}
+
+.field {
+  width: 100%;
+}
+.buttonsWrapper button {
```

Comment:
This is used in the example only. (We try to keep example/styleguide CSS  out of main component CSS by having separate ComponentName.example.css file). 

Another thing to note: we don't want to refer to elements anywhere except in marketplaceIndex.css file - add classes instead.